shared/params_test.go

```diff
@@ -693,3 +694,10 @@ func TestExtractRunIDsBodyParam_Replayable(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, payload, replayed)
 }
+
+func TestParseTestRunFilterParams_Page(t *testing.T) {
```

Comment:
Do we already have an implicit or explicit test for missing `page` value? If so, just this test LGTM.

Reply:
Yeah; all tests for TestRunFilterParams that don't set the `"page"` field